addons/sale/models/sale.py

```diff
@@ -47,8 +47,8 @@ def _amount_all(self):
                 amount_untaxed += line.price_subtotal
                 amount_tax += line.price_tax
             order.update({
-                'amount_untaxed': order.pricelist_id.currency_id.round(amount_untaxed),
-                'amount_tax': order.pricelist_id.currency_id.round(amount_tax),
+                'amount_untaxed': amount_untaxed,
+                'amount_tax': amount_tax,
```

Comment:
The rounding is done when setting the value in cache, right?
You should add a comment that explains this, so that someone else will not try to reintroduce rounding.

Reply:
It was already forwarded-ported